_posts/2019-01-30-get-started-libaktualizr.adoc

```diff
@@ -0,0 +1,264 @@
+= Get started
+:page-layout: page
+:page-categories: [using-libaktualizr]
+:page-date: 2018-11-28 14:08:55
+:page-order: 2
+:icons: font
+
+Aktualizr is a {cpp} application that implements client-side functionality for
+HERE OTA Connect system based on the Uptane security framework<<anchor-1,[1]>>.
+While it is possible to use Aktualizr as a standalone CLI application,
+using it as a {cpp} library from a custom application provides user with
+more flexibility and customization.
+
+This guide focuses on integrating Aktualizr functionality into a
+custom application, in other words it focuses on using libaktualizr.
+
+== How to build libaktualizr
+
+Aktualizr is an open-source software distributed
+under the Mozilla Public License 2.0 and is located on github:
+https://github.com/advancedtelematic/aktualizr
+
+The project is intended to be build on a Linux system and compiler must support
+C++11 version of the standard.
+File README.adoc located in the project root directory contains
+extended description on how to build it as well as the list of
+build and run-time dependencies.
+Please, refer to this list and install all the dependencies before
+proceeding to the next step.
+
+These are the steps needed to locally build a Release version of libaktualizr:
+[source,bash]
+$ git clone --recursive https://github.com/advancedtelematic/aktualizr.git
+$ cd aktualizr
+$ mkdir build
+$ cd build
+$ cmake ..
```

Comment:
I would still advise using `-DCMAKE_BUILD_TYPE=Release` explicitly, not to rely too much on current defaults.

Reply:
makes sense